### ISSUE TITLE
dispValue field name not recognized during data acquisition

### ISSUE THREAD

**Issue by manish-alluri** [OpenNFT/OpenNFT#167]
Hi Yury and team,

We were trying to launch our new rtfmri project and were facing an issue during the data acquisition at the scanner. OpenNFT is able to setup and start the acquisition but as soon the first file is received, it is showing the below error which is something new and I wasn't able to resolve it after trying to debug it a few times. I have included the error message below if someone could help us resolving this issue that has never occurred in the earlier project. Do let me know if you need any additional information regarding this error.

2024-10-17 04:28:33 | INFO      | opennft:setup - Setup finished: 6.20 s
2024-10-17 04:28:35 | INFO      | opennft:start - *** Started ***
2024-10-17 04:28:35 | INFO      | opennft:startFilesystemWatching - Searching for *.nii in C:\db2\data\sub-99996\ses-01\opennft\rt_watch
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration - First file was reached
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration - Call iteration for file "RT-fMR_MB4_Video1_e1_d1.nii"
C:\db2\data\sub-99996\ses-01\opennft\rt_watch\RT-fMR_MB4_Video1_e1_d1.nii
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration -   setup after first volume: 0.12 s
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration -   preprocess signal: 2.00 ms
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration -   Drawings: 0.00 mcs
2024-10-17 04:28:57 | INFO      | opennft:main_loop_iteration - Elapsed time: 0.1480 s

=============
Matlab time stamp t6!
=============

Unrecognized field name "dispValue".

Error in mainLoopEntry (line 108)
    displayData.dispValue = mainLoopData.dispValue;

Traceback (most recent call last):
  File "c:\db2\code\opennft\opennft\opennft\opennft.py", line 677, in call_main_loop
    next(self.main_loop)
  File "c:\db2\code\opennft\opennft\opennft\opennft.py", line 696, in main_loop_iteration
    self.eng.mainLoopEntry(self.iteration, nargout=0)
  File "C:\db2\code\OpenNFT\opennft-venv\lib\site-packages\matlab\engine\matlabengine.py", line 70, in __call__
    return FutureResult(self._engine(), future, nargs, _stdout,
  File "C:\db2\code\OpenNFT\opennft-venv\lib\site-packages\matlab\engine\futureresult.py", line 67, in result
    return self.__future.result(timeout)
  File "C:\db2\code\OpenNFT\opennft-venv\lib\site-packages\matlab\engine\fevalfuture.py", line 82, in result
    self._result = pythonengine.getFEvalResult(self._future,self._nargout, None, out=self._out, err=self._err)
matlab.engine.MatlabExecutionError:
  File C:\db2\code\OpenNFT\OpenNFT\opennft\matlab\mainLoopEntry.m, line 108, in mainLoopEntry
Unrecognized field name "dispValue".

Thanks,
Manish

**Comment by ykoush** [comment]
@manish-alluri Please briefly let us know what was the problem and solution. Thank you

**Comment by manish-alluri** [comment]
Hi all,

I have encountered this issue after I have changed the timing configuration in the .json file of the subjects according to our study. It would not allow me to start a run with a nuisance interval in the timing file which was causing the issue as it was coded to look for the baseline interval to begin with. I was trying to see if I could find and solve this constraint of not always having to start with the baseline block from the code but found it a bit difficult to change or update the functionality. However, I was able to resolve this error by going back to starting with a baseline block in the timing file. 

I will try to work to find a better way to handle this situation in the code and find a solution to this error, it would be helpful if any of the devs can take a look if they get the time sometime later as they would know the modularization of this functionality better.

Thanks,
Manish
